.circleci/config.yml

```diff
@@ -41,16 +41,20 @@ workflows:
   version: 2
   main:
     jobs:
-      - build-linux
+      - build-test-linux
 
 jobs:
-  build-linux:
+  build-test-linux:
     executor: linux
     steps:
       - setup
-      - run:
-          name: "build lotus-soup"
-          command: pushd lotus-soup && go build -tags=testground .
+      #- run:
```

Comment:
or could shunt it to a separate job that we don't wait for?

Reply:
Yeah, let's not disable the build for the lotus-soup test plan entirely.